transaction/serializers.py

```diff
@@ -1,8 +1,47 @@
 from rest_framework import serializers
-from .models import Transaction
+from base.serializers import LiquidatorSerializer
+from .models import Transaction, RecurringTransaction, TransactionTemplate
 
 
-class TransactionSerializer(serializers.ModelSerializer):
+class TransactionSerializer(LiquidatorSerializer, serializers.ModelSerializer):
```

Comment:
LiquidatorSerializer inherits from ModelSerializer, so there is no need to inherit from both